apiserver/facades/agent/provisioner/provisioner.go

```diff
@@ -1045,7 +1059,9 @@ type hostChangesContext struct {
 	result params.HostNetworkChangeResults
 }
 
-func (ctx *hostChangesContext) ProcessOneContainer(env environs.Environ, callContext context.ProviderCallContext, idx int, host, container *state.Machine) error {
+func (ctx *hostChangesContext) ProcessOneContainer(
+	env environs.Environ, callContext context.ProviderCallContext, idx int, host, container Machine,
```

Comment:
odd to me that you changed the other variable away from "container", but then use the word "container" here.

Reply:
I didn't "use" it, but nor did I change it :)
Will address consistency issue.